qoffer-infrastructure/src/main/groovy/life/qbic/portal/portlet/qOfferPortlet.groovy

```diff
@@ -12,8 +12,10 @@ import groovy.transform.CompileStatic
 import groovy.util.logging.Log4j2
 
 /**
- * Entry point for portlet qOffer_2.0. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
+ * Entry point for portlet qoffer_2.0. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
  *
+ * @since: 1.0
+ * @author: Jennifer Bödker
  * @see <a href=https://github.com/qbicsoftware/portal-utils-lib>portal-utils-lib</a>
```

Comment:
Why include the `@see` here?

Reply:
This was autogenerated by qube to better the elucidate the portlet structure and environment at QBiC

Reply:
Nit:
I am not sure whether this is up-to-date or not. We can leave it in but I would suggest removing it here.